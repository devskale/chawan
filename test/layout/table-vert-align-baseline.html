<!DOCTYPE html>
<table>
<tr>
<td style="vertical-align: baseline">
test
</td>
<td style="vertical-align: baseline">
test<br>
test
</table>
<div style="display: table">
<div style="display: table-cell">
<div style="width: 1ch; height: 1em; background: red"></div>
</div>
<div style="display: table-cell">test</div>
</div>
<br>
<br>
<div style="display: table">
<div style="display: table-cell">
<div>
<div style="width: 1ch; height: 1em; background: red; margin-top: -2em"></div>
</div>
</div>
<div style="display: table-cell">test</div>
</div>
<div style="display: table">
<div style="display: table-cell">
<div>
<div style="background: red; display: flow-root; height: 3em"></div>
test
<div>test</div>
<div>test</div>
</div>
</div>
<div style="display: table-cell">test</div>
</div>
