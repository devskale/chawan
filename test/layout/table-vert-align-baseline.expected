 test test
      test
[48;2;255;0;0m [49mtest

[48;2;255;0;0m [49m
 test
[48;2;255;0;0m    [49m
[48;2;255;0;0m    [49m
[48;2;255;0;0m    [49m
testtest
test
test
